Updated Readme File

AnotherNewName opened the pull request:
Updated the Readme file to properly reflect the controls for PC, since, previously, the only way to figure out the PC controls was either trial and error, or looking through the code to check for the specific keys being checked for.

AnotherNewName committed:
Updated Readme File

Updated the Readme file to properly reflect the controls for PC, since, previously, the only way to figure out the PC controls was either trial and error, or looking through the code to check for the specific keys being checked for.

omckeon commented:
![](https://github.trello.services/images/mini-trello-icon.png) [DXBallGame - add/fix documentation/explanation relating to game controls on PC](https://trello.com/c/w7C4FCPN/377-dxballgame-add-fix-documentation-explanation-relating-to-game-controls-on-pc)

ayanmasood commented:
New controls seem to be correct. Just a few suggestions on improving the layout. The controls might look better if separated and be further at the top. There are formatting issues with the controls section (no gap between point a. b. c.)Best to use dot points instead of letters. Also backticks should be used for code snippets (instead of bold).

ayanmasood reviewed (commented):
New controls seem to be correct. Just a few suggestions on improving the layout. The controls might look better if separated and be further at the top. There are formatting issues with the controls section (no gap between point a. b. c.)Best to use dot points instead of letters. Also backticks should be used for code snippets (instead of bold).

AnotherNewName committed:
Updating Readme file

Changed controls explanation to be an actual list, and replaced bold with backticks for code snippets

AnotherNewName commented:
Updated the readme file to implement the changes requested.

omckeon reviewed (approved):
Looks good.
